features/other_a/languages.feature

```diff
@@ -1,5 +1,32 @@
 @works
 Feature: Languages
+
+  Scenario: Add a language and check abuse and support available
```

Comment:
If you want to test just the basic process of creating a language, like the "Admin set up the language" and "TODO: Then I should be able to add a language in the front end" comments suggests we might want...

```
Scenario: An admin can add a language

Given basic languages
  And I am logged in as an admin
When I go to the languages page
  And I follow "Add a Language"
  And I fill in "Name" with "Klingon"
  And I fill in "Abbreviation" with "tlh"
  And I press "Create Language"
Then I should see "Language was successfully added."
  And I should see "The Archive supports these languages"
  And I should see "Klingon"
```